drivers/unix/os_unix.h

```diff
@@ -53,7 +53,7 @@ class OS_Unix : public OS {
 
 	virtual String get_stdin_string(bool p_block) override;
 
-	virtual Error get_entropy(uint8_t *r_buffer, int p_bytes) override; // Should return cryptographycally-safe random bytes.
```

Comment:
Just removed the comment as it was already in `core/os/os.h` and other implementations don't repeat it.